spec/features/pictures_feature_spec.rb

```diff
@@ -0,0 +1,118 @@
+require 'rails_helper'
+
+feature 'Pictures' do
+
+  before(:each) do
+    @user = create(:user)
+    @user2 = create(:user, username:'some_user', email: 'test2@email.com')
+  end
+
+  context 'no pictures have been uploaded' do
+
+    scenario 'should have link to upload a picture' do
+      visit '/pictures'
+      expect(page).to have_content "No one has uploaded any pictures yet!"
+    end
+
+  end
+
+  context 'pictures has been uploaded' do
+
+    scenario 'display pictures' do
+      @user.pictures.create(caption: 'coding at home', name: 'Nightlife', image_file_name: 'nightlife.png')
```

Comment:
Metrics/LineLength: Line is too long. [107/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)